collectors/python.d.plugin/mysql/mysql.chart.py

```diff
@@ -612,10 +660,40 @@ def _get_data(self):
             else:
                 self.queries.pop('slave_status')
 
+        if 'user_statistics' in raw_data:
+            if raw_data['user_statistics'][0]:
+                userstats_vars = [e[0] for e in raw_data['user_statistics'][1]]
+                for i in range(len(raw_data['user_statistics'][0])):
```

Comment:
use `enumerate`
`for i, _ in enumerate(raw_data['user_statistics'][0]):`

Reply:
fixed